src/lib/es2015.proxy.d.ts

```diff
@@ -1,18 +1,17 @@
 interface ProxyHandler<T extends object> {
     getPrototypeOf? (target: T): object | null;
-    setPrototypeOf? (target: T, v: any): boolean;
+    setPrototypeOf? (target: T, v: object | null): boolean;

```

Comment:
The&nbsp;**Proxy** exotic&nbsp;object’s [<code>[[SetPrototypeOf]]&nbsp;(&nbsp;<var>V</var>&nbsp;)</code>](https://tc39.es/ecma262/#sec-proxy-object-internal-methods-and-internal-slots-setprototypeof-v) internal&nbsp;method ensures&nbsp;that&nbsp;`V` is&nbsp;`object | null` before&nbsp;passing&nbsp;it to&nbsp;user&nbsp;code.